examples/js/loaders/EquiangularToCubeGenerator.js

```diff
@@ -0,0 +1,111 @@
+/**
+* @author Richard M. / https://github.com/richardmonette
+*/
+
+THREE.EquiangularToCubeGenerator = function( sourceTexture, resolution ) {
+
+	this.sourceTexture = sourceTexture;
+	this.resolution = resolution;
+
+ 	this.views = [
+		{ t: [1.0,  0.0,  0.0 ], u: [0.0, -1.0,  0.0] },
+		{ t: [-1.0,  0.0,  0.0], u: [0.0, -1.0,  0.0] },
+		{ t: [0.0,  1.0,  0.0],  u: [0.0,  0.0,  1.0] },
+		{ t: [0.0, -1.0,  0.0],  u: [0.0,  0.0, -1.0] },
+		{ t: [0.0,  0.0,  1.0],  u: [0.0, -1.0,  0.0] },
+		{ t: [0.0,  0.0, -1.0],  u: [0.0, -1.0,  0.0] },
+	];
+
+	this.camera = new THREE.PerspectiveCamera( 90, 1, 0.1, 10 );
+	this.planeMesh = new THREE.Mesh( new THREE.BoxBufferGeometry( 1, 1, 1 ), this.getShader() );
+	this.planeMesh.material.side = THREE.DoubleSide;
```

Comment:
Why is it `planeMesh` and why is it double-sided?

Reply:
`planeMesh` should be renamed to `boxMesh`, it's double sided because the camera is inside the box, i.e. it needs to render and not be backface culled.

Reply:
side: THREE.BackSide;

As you know, users copy code, so let's make it clear. :)

Reply:
👍(btw, appreciate you taking the time to work through the code with me!)